dm/worker/config.go

```diff
@@ -171,6 +171,10 @@ func (c *Config) adjust() error {
 		}
 	}
 
+	if c.Name == "" {
+		c.Name = c.AdvertiseAddr
```

Comment:
how about adding a log

Reply:
added in https://github.com/pingcap/dm/pull/461/commits/1ac078547d53799931886bf9b919b9deef321fa9